src/eko/output/manipulate.py

```diff
@@ -137,7 +137,9 @@ def flavor_reshape(
         inv_inputbasis = np.linalg.inv(inputbasis)
 
     # build new grid
-    for q2, elem in eko.items():
+    # for q2, elem in eko.items():
+    for q2 in eko.Q2grid:
+        elem = eko[q2]
```

Comment:
Wait, why not using `items()`?
(moreover, `Q2grid` here is not needed, since you only need to generate the array, `EKO` has already `__iter__` defined that does the exact same job)

Reply:
see [above](https://github.com/N3PDF/eko/pull/105#issuecomment-1190523057)